src/subcommand/wallet/restore.rs

```diff
@@ -45,6 +45,7 @@ impl Restore {
           name,
           settings,
           mnemonic.to_seed(self.passphrase.unwrap_or_default()),
+          bitcoincore_rpc::json::Timestamp::Time(0),
```

Comment:
Should we allow setting the timestamp for mnemonic restoration?

Reply:
I think we should, but this PR is an improvement, so I'd like to merge it right away and create a new issue. There are potentially considerations like should the user be able to set a date instead of a unix timestamp, etc.